UnlimitedAccess/TaskManager.swift

```diff
@@ -0,0 +1,14 @@
+//
+// Created by pavels.garklavs on 03/03/2022.
+//
+
+import Foundation
+
+final class TaskManager {
```

Comment:
let's rename it

Reply:
To what should I rename it?

Reply:
idk, but it isn't taskManager

Reply:
Okay, will think of something and rename it. Thanks!